hadoop/yarn/yarn_mutation.py

```diff
@@ -2,17 +2,28 @@
 from xml.etree import ElementTree as ET
 
 
-class YarnMutationConfig:
+class MutationBase(object):
 
     def __init__(self):
         self._xml = ElementTree(element=Element("sched-conf"))
-        self._update_queue = Element("update-queue")
-        self._xml.getroot().append(self._update_queue)
+
+    @property
+    def xml(self):
+        return self._xml
+
+    @xml.setter
+    def xml(self, value: ElementTree):
+        self._xml = value
+
+class YarnUpdateQueue(MutationBase):
 
     def add_queue(self, queue: str, **kwargs):
+        update_queue = Element("update-queue")
```

Comment:
get or update the existing element, so multiple add_queue can be issues on the same object

add a test case for this

Reply:
fixed